specification/semantic_conventions.md

```diff
@@ -183,3 +185,28 @@ provides a range of lines, the second `lineno` can be provided after a colon `:`
 - ` <col>` or ` <col>:<col>` - OPTIONAL - If the runtime provides a column or column range, it MAY be provided
 after the `lineno`, separated by a space.
 - literal `)`
+
+#### PPROF Profile.proto Data Format
+
+[`Profile.proto`](https://github.com/google/pprof/tree/master/proto) is a data representation for profiling data. It is
+independent of the type of data being collected and the sampling process used to collect that data.
+The log message will contain a gzip-compressed, base64-encoded protocol buffer conforming to `profile.proto`. Each message
+contains either `allocation` or `cpu` samples determined by the data type specified for the log record.
+
+For each `allocation` and `cpu` sample:
+- label `source.event.name` of type `string` OPTIONALLY can contain the name of the event that triggered the sampling
+- label `source.event.time` of type `long` MUST be set to the time when the sample was taken
+- label `trace_id` of type `string` MUST be set when sample was taken within a span scope
+- label `span_id` of type `string` MUST be set when sample was taken within a span scope
+- label `thread.id` of type `long` OPTIONALLY can be set to the thread identifier used by the runtime environment
```

Comment:
Is this referring to the application space or just user space of the kernel?

Reply:
the intention was that `thread.id` refers to applications specific thread id e.g. java has thread ids that aren't the same as os thread ids